includes/class-newspack-electionkit-settings.php

```diff
@@ -121,6 +141,18 @@ public static function newspack_electionkit_ballotpedia_api_key_callback() {
 		);
 	}
 
+	/**
+	 * Render Region Filter textbox.
+	 */
+	public static function newspack_electionkit_region_filter_callback() {
+		$newspack_electionkit_region_filter = get_option( 'newspack_electionkit_region_filter', false );
+		printf(
+			'<input type="text" id="newspack_electionkit_region_filter" name="newspack_electionkit_region_filter" aria-describedby="newspack_electionkit_region_filter-description" value="%s" class="regular-text"/><p class="description" id="newspack_electionkit_region_filter-description">%s</p>',
```

Comment:
If only one state can be selected, maybe use a  `select` e.g. https://www.freeformatter.com/usa-state-list-html-select.html